src/common/Widget.js

```diff
@@ -578,6 +579,29 @@
     };
 
     //  DOM/SVG Node Helpers  ---
+    
+    Widget.prototype.showBusy = function (show) {
+        if (show) {
+            //show the indicator
+        } else {
+            //hide the indicator
```

Comment:
sounds good ... it seems this project is moving towards also being a UI library ....